app/models/podcast_import.rb

```diff
@@ -126,12 +128,12 @@ def create_podcast
     podcast_attributes[:explicit] = feed.itunes_explicit
     podcast_attributes[:new_feed_url] = feed.itunes_new_feed_url
     podcast_attributes[:path] ||= feed.feedburner_name
+    podcast_attributes[:feedburner_url] ||= feed.feedburner_url
+    podcast_attributes[:url] ||= feed.feedburner_url
 
-    podcast_attributes[:author] = person(feed.itunes_author)
+    podcast_attributes[:author] = feed.author ? person(feed.author) : person(feed.itunes_author)
     podcast_attributes[:managing_editor] = person(feed.managing_editor)
-    podcast_attributes[:owners] = Array(feed.itunes_owners).map do |o|
-      { name: o.name, email: o.email }
-    end
+    podcast_attributes[:owner] = person(feed.owner)
```

Comment:
note this `person()` call is redundant since feed.owner will already be in the { name: name, email: email } format -- I left it in there for symmetry with the author and managing editor attribute settings, so someone glancing just at this will know that those 3 will all have the same structure.